caikit/core/model_management/local_model_finder.py

```diff
@@ -0,0 +1,54 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The LocalModelFinder locates models locally on disk
+"""
+# Standard
+from typing import Optional
+import os
+
+# First Party
+import aconfig
+import alog
+
+# Local
+from ..modules import ModuleConfig
+from ..toolkit.errors import error_handler
+from .model_finder_base import ModelFinderBase
+
+log = alog.use_channel("LFIND")
+error = error_handler.get(log)
+
+
+class LocalModelFinder(ModelFinderBase):
+    __doc__ = __doc__
```

Comment:
This is shorthand to avoid needing to have a docstring for the module and the class if they're essentially the same thing. The LHS is the attribute name on the class and the RHS references the module-level `__doc__`.

Reply:
Oh interesting. Thanks!